VARIABLES.md

```diff
@@ -1852,17 +1868,25 @@ Default:  1.6.2
 
 ### jolokia_jar_url
 
-Full URL used for Jolokia Agent Jar Download
+Full URL used for Jolokia Agent Jar Download. When `jolokia_url_remote=false` this represents the path on Ansible control host.
```

Comment:
@JumaX I meant these docs. Do you like the phrasing? 
I think we am cool w just doing docs here going forward, and for major things we do updates in the official docs

Reply:
Sure, sounds good.